packages/dao/contracts/Api3BaseTemplate.sol

```diff
@@ -112,20 +112,22 @@ contract Api3BaseTemplate is IsContract {
 
     /* AGENT */
```

Comment:
As a general comment. Why use a custom `BaseTemplate` instead of using the `@aragon/templates-shared` one ? Have you updated other methods than the `_installDefaultAgentApp` and `_installVotingApp` ones ? Or can I consider them safe ?

Reply:
I've removed all changes from Api3BaseTemplate because all of the work may be done in Api3Template... About the first question, I am not sure if those are the same contracts, as Api3BaseTemplate was created before we started the work o this project, but will add an issue to check this out and remove Api3BaseTemplate if needed..

Reply:
I think they are pretty much the same excepts for those two edited functions. But yeah, be worth checking.

Reply:
I had implemented this for ease of comparing
https://github.com/api3dao/api3-dao/blob/develop/packages/api3-voting/package.json#L6
A similar thing can be done here (if we even need a customized base template).